src/psycopmlutils/timeseriesflattener/flattened_dataset.py

```diff
@@ -289,30 +302,225 @@ def _validate_processed_arg_dicts(self, arg_dicts: list):
                 f"Didn't generate any features because: {warnings}",
             )
 
-    def _flatten_temporal_values_to_df_wrapper(self, kwargs_dict: Dict) -> DataFrame:
-        """Wrap flatten_temporal_values_to_df with kwargs for multithreading
-        pool.
+    def _get_feature(self, kwargs_dict: Dict) -> DataFrame:
+        """Get features. Either load from cache, or generate if necessary.
```

Comment:
Afraid this will fail if adding from a dataframe with multiple columns (e.g. text embeddings).

Reply:
Agreed, see top level comment.